### PR TITLE
ticket 12

### PR DESCRIPTION
**Pull request by Gazdean**

### CODE REVIEW

File: __tests__/api.test.js [hunk 1]
```diff
@@ -127,6 +127,15 @@ describe('GET /api/articles/:article_id', () => {
         expect(body.message).toBe('bad request');
         });
     })
+    test('the response objects has a comment_count property, which is the total number of comments for that article', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+            const commentCount = body.article.comment_count
```
Comment (B7Bprog) [inline __tests__/api.test.js:135]: Please check the other properties as well and the exact article id to make sure it's the right article with all it's properties present.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Gareth, just a little refactor with the test, then it's good to go.

File: __tests__/api.test.js (hunk 1 above)
Comment (Gazdean) [reply __tests__/api.test.js:135]: i was going to, but noticed my first test tests for those so deleted them
will add them back in
thanks

Comment (Gazdean) [reply __tests__/api.test.js:135]: done

**Comment by Gazdean** [comment]
merged
branch deleted

File: __tests__/api.test.js (hunk 1 above)
Comment (Gazdean) [reply __tests__/api.test.js:135]: apologies
i see
do this testing of properties as it could be a different article number

**Pull request merged**